guide/miscellaneous/making-an-eval-command.md

```diff
@@ -0,0 +1,123 @@
+# Making an Eval command
+
+## Introduction
+
+### What is eval?
+
+The `eval()` function evaluates JavaScript code represented as a string.
+For instance, `eval(1 + 1)` will return `2`. Similarly, `eval(client.user.tag)` returns the bot's tag, e.g. `MyBot#0001`.
+
+### Why eval can be dangerous?
+
+:::danger
+`eval()` is a dangerous function, which executes the supplied code, with the privileges of the caller. If you run `eval()` with a string that could be affected by a malicious party, you may end up running malicious code on your machine. More importantly, third-party code can see the scope in which `eval()` was invoked, which can lead to possible attacks.
+
+In other words, eval is dangerous if somehow someone else has access to it, the person can run (malicious) code with administrator access.
+:::
+
+:::tip
+`eval()` is also slower than the alternatives since it has to invoke the JS interpreter, while many other constructs are optimized by modern JS engines.
+
+::: tip
+`eval()` is also slower than the alternatives since it has to invoke the JS interpreter, while many other constructs are optimized by modern JS engines.
```

Comment:
I think you have an accidental duplication there

Reply:
Thanks for the suggestion, @Darqam!